src/components/ActionsBuilder/Option/Option.tsx

```diff
@@ -106,12 +106,11 @@ export const OptionRow: React.FC<OptionRowProps> = ({
 
       <ActionsWrapper indented={isEditable}>
         {!isEditable &&
-          option?.actions?.map((action, index) => {
-            const permissionArgs = option?.permissions?.[index];
+          option?.decodedActions?.map((action, index) => {
+            const permissionArgs = option?.permissions[index];
             return (
               <ActionRow
                 key={index}
-                call={action}

```

Comment:
I think `ActionRow` is expecting the call, and `Call` =! `DecodedAction`. Please verify that by changing `options.actions` to `options.decodedActions` you are not braking sub-components. 
You can dismiss this if you are sure of it.